Build failure on 0.9.1-SNAPSHOT

#### Issue Description

Build fails with 
![](https://files.gitter.im/audreyloeffel/bwsF/blob)

```

[ERROR] Failed to execute goal on project nd4j-api: Could not resolve dependencies for project org.nd4j:nd4j-api:jar:0.9.1-SNAPSHOT: Failure to find org.nd4j:jackson:jar:0.9.1-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of sonatype-nexus-snapshots has elapsed or updates are forced -> [Help 1]
```

And indeed nd4j-api is before nd4j-jackson in the reactor (this is new, jackson used to precede)

Bisection shows:

```
huitseeker@pegasus➜huitseeker/DL4J/nd4j(1bbc3e7b9✗)» git bisect bad  [17:34:52]
1bbc3e7b9631a2a1f95b69920ba186adedb586ed is the first bad commit
commit 1bbc3e7b9631a2a1f95b69920ba186adedb586ed
Author: Adam Gibson <adam@skymind.io>
Date:   Thu Aug 3 18:54:41 2017 +0900

    new indexing test and fix for new axis at beginning (WIP)

:040000 040000 07cdc05322ad76ae5b3832314c665a3c570b14ca 081803b4cd0ec6c383498342344396ce931fb63d M	nd4j-backends
:100644 100644 4d58026299efda22ec75feda584533238ab6e0a2 5acf089e82a5278fe66f160882f566a3703d4e95 M	pom.xml
```

--- a/buildmultiplescalaversions.sh
+++ b/buildmultiplescalaversions.sh
@@ -1,5 +1,5 @@
 #! /bin/bash
-BASEDIR=$(dirname $(readlink -f "$0"))
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 function echoError() {
     (>&2 echo "$1")

